### PR TITLE
`Test`: Add more e2e exam participation and file upload exam exercise tests 

### PR DESCRIPTION
**Pull request by TheZoker**
#### General

- [x] This is a small issue that I tested locally
- [x] Language: I followed the [guidelines for inclusive, diversity-sensitive, and appreciative language](https://docs.artemis.ase.in.tum.de/dev/guidelines/language-guidelines/).
- [x] I chose a title conforming to the [naming conventions for pull requests](https://artemis-platform.readthedocs.io/en/latest/dev/guidelines/development-process.html#naming-conventions-for-github-pull-requests).

### Motivation and Context

Currently the exam student participation is only tested using one student with one submission. Since exams usually have more than one participant and these participants have different submissions, this should be improved. Also the file upload exercise type tests are missing from the exam mode. 

### Description

This PR adds two more students to the exam participation list and creates different submission with different level of correctness to the existing e2e tests. It also adds logic for file upload exam exercises, since those were not tested until now. In the future, these tests can be extended to have even more students and even much more different submissions (maybe even random solutions).

### Steps for Testing

1. Run the test against your local system by adjusting the `cypress.json` file within the `src/test/cypress` folder and using the `npm run cypress:open` command
2. Then a cpyress instance will be launched, which allows to run manual test suites against the local artemis instance. 

Alternatively have a look at the run within bamboo:
https://bamboo.ase.in.tum.de/browse/ARTEMIS-AETG845-6

#### Code Review
- [ ] Review 1
- [ ] Review 2
#### Manual Tests
- [ ] Test 1
- [ ] Test 2

### Test Coverage

| Class/File | Branch | Line |
|------------|-------:|-----:|

### CODE REVIEW

**Comment by TheZoker** [comment]
Bump (to avoid stale)

File: src/test/cypress/package.json [hunk 1]
```diff
@@ -10,7 +10,7 @@
         "@4tw/cypress-drag-drop": "2.1.0",
         "@types/node": "17.0.24",
         "cypress": "9.5.4",
-        "cypress-file-upload": "5.0.8",
+        "cypress-file-upload": "^5.0.8",
```
Comment (ge65cer) [inline src/test/cypress/package.json:13]: Please use a fixed version.

**Review by ge65cer — COMMENTED** [review]
Looks good, the tests passed. One comment...

File: src/test/cypress/package.json (hunk 1 above)
Comment (TheZoker) [reply src/test/cypress/package.json]: Oh must have overlooked this. Thanks!

**Review by ge65cer — COMMENTED** [review]
Latest Cypress run failed, I triggered a re-test. Maybe you can take a closer look.

**Pull request closed without merging**